lib/plugins/aws/customResources/resources/eventBridge/handler.js

```diff
@@ -60,6 +60,8 @@ async function create(event, context) {
         input: EventBridgeConfig.Input,
         inputPath: EventBridgeConfig.InputPath,
         inputTransformer: EventBridgeConfig.InputTransformer,
+        retryPolicy: EventBridgeConfig.RetryPolicy,
```

Comment:
I'm not sure about adding support for these in custom resource while we currently have support for native CloudFormation resources for EventBridge. 

@medikoo What do you think?

Reply:
I would vote in favour of not extending deprecated (?) non-CFN way

Reply:
Yes, it's been deprecated and is staged to be removed with next major - it still uses CloudFormation, but relies on a custom resource which is problematic and was introduced due to lack of "native" CloudFormation support